templates/repo/scripts/Makefile

```diff
@@ -0,0 +1,44 @@
+# Auto-generated by fogg. Do not edit
+# Make improvements in fogg, so that everyone can benefit.
+OS:=$(shell uname -s | tr '[:upper:]' '[:lower:]')
+ifeq ($(shell uname -m),x86_64)
+  ARCH ?= "amd64"
+endif
+ifeq ($(shell uname -m),i686)
+  ARCH ?= "386"
+endif
+ifeq ($(shell uname -m),aarch64)
+  ARCH ?= "arm"
+endif
+
+TF_PRESENT := $(shell command -v terraform 2> /dev/null)
+TF_CURRENT_VERSION := $(shell terraform version | head -n1 | sed 's/Terraform v//g')
+TF_VERSION := 0.11.8
```

Comment:
this needs to come from a template var

Reply:
Oh yeah, pr still needs a bunch of work. Opened to get eyes on general approach